homework-submission/myapp/src/Category.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Items from './Items';
```

Comment:
Give this better naming like, `DATA, or ITEMS_DATA` .. usually **static data and constants** have names all with Capital letters.